<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(178, 52, 255);
font: 75 18pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>昵称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nickname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(178, 52, 255);
font: 75 18pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>输入密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passwd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passrewd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(178, 52, 255);
font: 75 18pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_phonenumber">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(178, 52, 255);
font: 75 18pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>手机号码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_phoneverification">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(178, 52, 255);
font: 75 18pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(25, 25, 25);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_verification">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 54, 3);</string>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
